lib/web_ui/lib/src/engine.dart

```diff
@@ -31,487 +33,177 @@ import 'dart:js_util' as js_util;
 import 'dart:math' as math;
 // ignore: unused_import
 import 'dart:svg' as svg;
+// ignore: unused_import
 import 'dart:typed_data';
-
+// ignore: unused_import
 import 'package:js/js.dart';
 // ignore: unused_import
 import 'package:meta/meta.dart';
-
+// ignore: unused_import
 import '../ui.dart' as ui;
-
 // ignore: unused_import
 import 'engine/configuration.dart';
+// ignore: unused_import
 import 'engine/embedder.dart';
+// ignore: unused_import
+import 'engine/initialization.dart';
+// ignore: unused_import
 import 'engine/keyboard.dart';
+// ignore: unused_import
 import 'engine/mouse_cursor.dart';
+// ignore: unused_import
 import 'engine/navigation/js_url_strategy.dart';
+// ignore: unused_import
 import 'engine/navigation/url_strategy.dart';
+// ignore: unused_import
 import 'engine/platform_dispatcher.dart';
+// ignore: unused_import
 import 'engine/platform_views/content_manager.dart';
+// ignore: unused_import
 import 'engine/profiler.dart';
-import 'engine/web_experiments.dart';
+// ignore: unused_import
+import 'engine/safe_browser_api.dart';
```

Comment:
Do you actually need these imports here? Now that you removed all code from this file, I think we can get rid of these unused `engine/*` imports.

Reply:
I thought these imports were needed because when the package is rewritten into one library with part files, these imports were used by the part files, no?

Reply:
Imports are only needed for things used in this file. Otherwise the analyzer will scream at you.

The sdk_rewriter deletes these imports, and it converts the exports into part directives: https://github.com/flutter/engine/blob/5ad06c2130f50910171208d5a68c6205018b969d/web_sdk/sdk_rewriter.dart#L49-L55

Reply:
Reminder to delete these imports if the build/tests pass without them.

Reply:
Ah, sounds good. Deleting.

Reply:
Done.